mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -0,0 +1,28 @@
+use crate::bridge::NestedBridge;
+use crate::settings::Settings;
+use tracing::info;
+
+#[derive(Default)]
+pub struct BridgeController {
```

Comment:
code comments would be great. You can even reference bridge design doc here.

Reply:
I think design doc is internal, probably can not be accessed by external people